lib/routes/highwaterApi.js

```diff
@@ -38,6 +38,24 @@ module.exports = function (userApiClient, kissClient, salt) {
     return hash.finalize().toString().substr(0, len);
   }
 
+  function sendMetrics(req, user, hash = true) {
```

Comment:
the hash parameter isn't used in the function or it's an omission to hash user param (and maybe the req.params.patientID added).

Reply:
it is intentional but I'm still wondering if I should hash it or not. 
The userId hash is done when sending the data to KissMetrics. 
By the way it's answering my question: if we send the data to Sumologic, we have to hash it.